src/components/Method/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Index = ({ method: { title, summary } }) => (
```

Comment:
Can we change `Index` const to `Method` here so it's clearer?

Reply:
Oh damn my bad, of course!